assembly/gradle-plugin/src/test/java/org/eclipse/sw360/antenna/frontend/gradle/AntennaGradlePluginTest.java

```diff
@@ -118,4 +120,27 @@ public void testWithoutGradleSetSystemEnvironmentVariables() throws AntennaExcep
 
         assertThat(root.resolve("antenna")).exists();
     }
+
+    @Test
+    public void testWithPomInSubDirs() throws IOException, AntennaException {
+        Path pomParentPath = exampleTestProject.getProjectPom().getParent().normalize();
+        Path dest = Paths.get(pomParentPath.toString(), "src", "pom.xml");
+        Files.move(exampleTestProject.getProjectPom(), dest, REPLACE_EXISTING);
+
+        String buildGradle = "plugins {\nid 'org.eclipse.sw360.antenna'\n}\n" +
+                "AntennaConfiguration{\ntoolConfigurationPath '" + dest + "'\n}";
+        FileUtils.writeStringToFile(projectRoot.resolve("build.gradle").toFile(), buildGradle, StandardCharsets.UTF_8);
+
+        AntennaImpl runner = new AntennaImpl("antenna-maven-plugin",
```

Comment:
This is duplicated code to the test above, maybe extract it into a method or create parameterized tests